### PR TITLE
Update index.ts

### PR DESCRIPTION
**Pull request by JingMu457**
user name: nl
ip addres:223.111.148.195

### CODE REVIEW

**Review by JingMu457 — COMMENTED** [review]
用户名：nl
ip:223.111.148.195

**Comment by LowEntropyBody** [comment]
Congratulations on passing the proposal. Please resolve conflicts with main branch. After that, we will megre PR and your Gateways will be online.

File: packages/apps-config/src/ipfs-gateway-endpoints/index.ts [hunk 1]
```diff
@@ -86,10 +86,11 @@ export function createAuthIpfsEndpoints (t: TFunction): AuthIpfsEndpoint[] {
...
-    //   location: t('Beijing'),
-    //   text: t('️Deklod'),
-    //   value: 'https://ipfs-gw.dkskcloud.com'
-    // }
+    {
+      location: t('Shanghai')，
+      text: t('Area51-GW'),
+      value: 'http://223.111.148.195'，
+      // crustAddress: cTHa2wtCS1LouRceECULTYCvdoi9HSMr638Uzx8zWsVP9mVDi
+      // contactInformation: jingmu.lemon@gmail.com
```
Comment (LowEntropyBody) [inline packages/apps-config/src/ipfs-gateway-endpoints/index.ts:94]: Syntax error, this is missing a parenthesis

File: packages/apps-config/src/ipfs-gateway-endpoints/index.ts [hunk 2]
```diff
@@ -86,10 +86,12 @@ export function createAuthIpfsEndpoints (t: TFunction): AuthIpfsEndpoint[] {
       status: 'online'
       //crustAddress: cTJUpCMAEzcLzaQej1A456S9v1qYpjFd8JgK9GLoeh4CKtaj7
     }
```
Comment (LowEntropyBody) [inline packages/apps-config/src/ipfs-gateway-endpoints/index.ts:88]: line 88, syntax error, missing a comma

**Pull request merged**